prow/jobs/control-plane/components/kyma-environment-broker/kyma-environment-broker-build.yaml

```diff
@@ -65,7 +65,8 @@ presubmits: # runs on PRs
             args:
               - "--name=control-plane/kyma-environments-cleanup-job"
               - "--context=components/kyma-environment-broker"
-              - "--dockerfile=Dockerfile.cleanup"
+              - "--dockerfile=Dockerfile.job"
+              - "--build-arg=BIN=environmentscleanup"
```

Comment:
I don't think there is a `build-arg` flag.
 - The closest thing is still in PR: #6733

Reply:
thanks! I was under the impression it's raw kaniko building the images. I will put this as WIP and keep an eye on https://github.com/kyma-project/test-infra/pull/6733

Reply:
@wozniakjan not really. It's a wrapper for kaniko or BuildKit. image-builder doesn't support passing build args, however you can try to add them to the environment variables through the env-file. Look into https://github.com/kyma-project/test-infra/blob/88ddcc0f22eee0249af560c4647afb92aa15e7ba/development/image-builder/main.go#L376 and refer to the README.md file https://github.com/kyma-project/test-infra/blob/main/development/image-builder/README.md

Or you can use something else for building go-based containers.

Reply:
given https://github.com/kyma-project/test-infra/pull/6733 merged, @ralikio is going to take over this and change `--build-arg=` to -`-export-tags=`